Jenkinsfile

```diff
@@ -2,7 +2,7 @@
 
 properties([
     parameters([
-        string(defaultValue: 'ci40', description: 'OpenWrt branch to test against', \
+        string(defaultValue: 'ci40', description: 'OpenWrt branch to test against',
```

Comment:
Plus this fix is still in testing in a single feed and shouldn't be added to all feeds yet so just drop the commit. It hasn't fixed anything the builds have started working because its only the first build per branch that fails.